### DISCUSSION TITLE
Timestamps like in ElevenLabs

### DISCUSSION THREAD

**Discussion by tmplogic** [fishaudio/fish-speech#717] [Q&A]
Does fish.audio support character-level timestamps like in ElevenLabs' /v1/text-to-speech/{voice_id}/with-timestamps endpoint?

**Comment by cod3r0k** [comment]
Do you have any update ? @tmplogic

**Reply by tmplogic** [reply]
No update yet sorry, and I haven't had the chance to check the actual output of their api to see if there is any timestamp info.

**Comment by fahimanwer** [comment]
Any update yet?
